src/main/java/org/jboss/xavier/integrations/jpa/service/FlagAssessmentService.java

```diff
@@ -1,5 +1,7 @@
 package org.jboss.xavier.integrations.jpa.service;
 
+import java.util.List;
```

Comment:
Changes to this class are not needed I think.

Reply:
+1